src/docs/sphinx/advancedExamples/validationStudies/mccWellbore/Example.rst

```diff
@@ -0,0 +1,235 @@
+
+We simulate a drained wellbore problem subjected to isotropic horizontal stress (:math:`\sigma_h`) and vertical stress (:math:`\sigma_v`), as shown below. By increasing the wellbore supporting pressure (:math:`P_w`), the wellbore expands, and the formation rock experiences elastoplastic deformation. A plastic zone develops in the near wellbore region.
+
+
+.. _problemSketchFig:
+.. figure:: MCC_wellSketch.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Sketch of the wellbore problem 
+
+
+To simulate this phenomenon, the Modified Cam-Clay model is used in this example. Displacement and stress fields around the wellbore are numerically calculated. These numerical predictions are then compared with the corresponding analytical solutions `(Chen and Abousleiman, 2013)  <https://www.icevirtuallibrary.com/doi/10.1680/geot.11.P.088>`__ from the literature. 
+
+
+
+In this example, we focus our attention on the ``Mesh`` tags,
+the ``Constitutive`` tags, and the ``FieldSpecifications`` tags.
+
+------------------------------------------------------------------
+Mesh
+------------------------------------------------------------------
+
+Following figure shows the generated mesh that is used for solving this wellbore problem.
+
+
+.. _problemSketchFig:
+.. figure:: MCC_wellMesh.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Generated mesh for the wellbore problem
+
+
+Let us take a closer look at the geometry of this wellbore problem.
+We use the internal mesh generator ``InternalWellbore`` to create a rock domain
+(:math:`10\, m \, \times 5 \,  m \, \times 2 \, m`), with a wellbore of
+initial radius equal to :math:`0.1` m.
+Coordinates of ``trajectory`` defines the wellbore trajectory, which represents a vertical well in this example. 
+By turning on ``autoSpaceRadialElems="{ 1 }"``, the internal mesh generator automatically sets number and spacing of elements in the radial direction, which overrides the values of ``nr``. 
+With ``useCartesianOuterBoundary="0"``, a Cartesian aligned outer boundary on the outer block is enforced.
+In this way, a structured three-dimensional mesh is created with 50 x 40 x 2 elements in the radial, tangential and z directions, respectively. All the elements are eight-node hexahedral elements (``C3D8``) and refinement is performed
+to conform with the wellbore geometry. This mesh is defined as a cell block with the name
+``cb1``.
+
+
+.. literalinclude:: ../../../../../../src/coreComponents/physicsSolvers/solidMechanics/benchmarks/PlasticWell_Base.xml
+    :language: xml
+    :start-after: <!-- SPHINX_WELLBORE_MESH -->
+    :end-before: <!-- SPHINX_WELLBORE_MESH_END -->
+
+
+------------------------
+Solid mechanics solver
+------------------------
+
+For the drained wellbore problem, the pore pressure variation is omitted. Therefore, we just need to define a solid mechanics solver, which is called ``mechanicsSolver``. 
```

Comment:
```suggestion
For this undrained wellbore problem, the pore pressure variation is omitted. Therefore, we just need to define a solid mechanics solver, which is called ``mechanicsSolver``. 
```

Reply:
see above comment about the drained condition.